lib/_http_outgoing.js

```diff
@@ -763,7 +763,9 @@ OutgoingMessage.prototype.end = function end(chunk, encoding, callback) {
   }
 
   if (chunk) {
-    if (typeof chunk !== 'string' && !(chunk instanceof Buffer)) {
+    if (typeof chunk !== 'string' &&
+      !(chunk instanceof Buffer) &&
+      !(chunk instanceof Uint8Array)) {
```

Comment:
You probably want to use `isUint8Array` from `internal/util/types` instead (if you search the source tree for `isUint8Array` there should be other examples), and you can drop the `instanceof Buffer` check because all `Buffer`s are `Uint8Array`s (we do usually list both types in the error message explicitly because not everybody is aware of that).

Reply:
@addaleax Done.